README.md

```diff
@@ -9,9 +9,11 @@ GOV.UK Frontend ·
 
 GOV.UK Frontend contains the code you need to start building a user interface for government platforms and services.
 
+The guidance and demos of GOV.UK Frontend content live in the [GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital/).
+
 ## Before using GOV.UK Frontend
 
-You should start by familiarising yourself with the [GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital/), which contains guidance and examples of components, styles and patterns you can use to design  government platforms and services.
+You should start by familiarising yourself with the [GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital/), which contains the guidance and examples of components, styles and patterns of the code that lives in GOV.UK Frontend.
```

Comment:
I don't think this is quite right - specifically 'patterns of the code that lives in GOV.UK Frontend'.

Reply:
Sure. @amyhupe could you please review the wording. I was trying to make the relationship of FE and DS more explicit.

Reply:
How about:

See live examples of GOV.UK Frontend components, and guidance on when to use them in your service, in the [GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital/).